### PR TITLE
fix: corrects hanging on white screen at startup

### PR DESCRIPTION
**Pull request by sanyappc**
* fix: corrects hanging on white screen at startup

### CODE REVIEW

**Comment by sanyappc** [comment]
#5786

File: app/stacks/OutsideStack.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const _OutsideStack = () => {
 	return (
 		<Outside.Navigator screenOptions={{ ...defaultHeader, ...themedHeader(theme), ...StackAnimation } as StackNavigationOptions}>
 			{/* @ts-ignore */}
-			<Outside.Screen name='NewServerView' component={NewServerView} options={NewServerView.navigationOptions} />
+			{/* <Outside.Screen name='NewServerView' component={NewServerView} options={NewServerView.navigationOptions} /> */}
```
Comment (GleidsonDaniel) [inline app/stacks/OutsideStack.tsx:26]:
```suggestion

```

**Review by GleidsonDaniel — CHANGES_REQUESTED** [review]

File: app/stacks/OutsideStack.tsx (hunk 1 above)
Comment (diegolmello) [reply app/stacks/OutsideStack.tsx:26]: Commenting is ok

**Review by diegolmello — APPROVED** [review]

**Comment by diegolmello** [comment]
Merged. Thanks for your contribution, @sanyappc!

**Pull request merged**